apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -169,4 +169,33 @@ export class BookingsRepository_2024_08_13 {
       },
     });
   }
+
+  async getUserBookingsForTimeRange(userId: number, startTime: Date, endTime: Date) {
+    return this.dbRead.prisma.booking.findFirst({
+      where: {
+        userId,
+        status: "ACCEPTED",
+        startTime: {
+          gte: startTime,
+          lt: endTime,
+        },
+      },
+    });
+  }
+
+  async getBookingsWhereUserIsAttendee(userEmail: string, startTime: Date, endTime: Date) {
```

Comment:
The function is called `getBookingsWhereUserIsAttendee` but has required `startTime` and `endTime` parameters - what if someone wants to get all bookings where user is an attendee?

I would rather call it `getBookingsAsAttendeeWithinTimeRange`.